spec/ioki/operator_api_spec.rb

```diff
@@ -111,4 +111,116 @@
         to eq(Ioki::Model::Operator::Vehicle.new)
     end
   end
+
+  describe '#drivers(product_id)' do
+    it 'calls request on the client with expected params' do
+      expect(operator_client).to receive(:request) do |params|
+        expect(params[:url].to_s).to eq('operator/products/0815/drivers')
+        result_with_index_data
+      end
+
+      expect(operator_client.drivers('0815', options)).
+        to eq([Ioki::Model::Operator::Driver.new])
+    end
+  end
+
+  describe '#driver(product_id, driver_id)' do
+    it 'calls request on the client with expected params' do
+      expect(operator_client).to receive(:request) do |params|
+        expect(params[:url].to_s).to eq('operator/products/0815/drivers/4711')
+        [result_with_data, full_response]
+      end
+
+      expect(operator_client.driver('0815', '4711', options)).
+        to eq(Ioki::Model::Operator::Driver.new)
+    end
+  end
+
+  describe '#create_driver(product_id, driver)' do
+    let(:driver) { Ioki::Model::Operator::Driver.new({ id: '4711' }) }
+
+    it 'calls request on the client with expected params' do
+      expect(operator_client).to receive(:request) do |params|
+        expect(params[:url].to_s).to eq('operator/products/0815/drivers')
+        [result_with_data, full_response]
+      end
+
+      expect(operator_client.create_driver('0815', driver, options)).
+        to eq(Ioki::Model::Operator::Driver.new)
+    end
+  end
+
+  describe '#update_driver(product_id, driver)' do
+    let(:driver) { Ioki::Model::Operator::Driver.new({ id: '4711' }) }
+
+    it 'calls request on the client with expected params' do
+      expect(operator_client).to receive(:request) do |params|
+        expect(params[:url].to_s).to eq('operator/products/0815/drivers/4711')
+        [result_with_data, full_response]
+      end
+
+      expect(operator_client.update_driver('0815', driver, options)).
+        to eq(Ioki::Model::Operator::Driver.new)
+    end
+  end
+
+  describe '#delete_driver(product_id, driver_id)' do
+    it 'calls request on the client with expected params' do
+      expect(operator_client).to receive(:request) do |params|
+        expect(params[:url].to_s).to eq('operator/products/0815/drivers/4711')
+        result_with_data
+      end
+
+      expect(operator_client.delete_driver('0815', '4711', options)).
+        to eq(Ioki::Model::Operator::Driver.new)
+    end
+  end
```

Comment:
hi there, sorry, but shouldn't this check for method as well, like the one below does for `lock`?
something like `expect(params[:method]).to eq(:delete)`?

Reply:
valid point. I've added the missing specs (also for the vehicle endpoints). Thanks 🙏